### PR TITLE
Use new dataflow pipeline deploy action

### PR DESCRIPTION
**Pull request by lock14**
* converted artificat pipeline autopush deployment to use the new action
* converted commit review status pipeline autopush deployment to use the new action

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -288,127 +288,50 @@ jobs:
...
-        name: 'Authenticate to Google Cloud'
-        uses: 'google-github-actions/auth@35b0e87d162680511bf346c299f71c9c5c379033' # ratchet:google-github-actions/auth@v1
-        with:
-          workload_identity_provider: '${{ vars.WIF_PROVIDER }}'
-          service_account: '${{ vars.WIF_SERVICE_ACCOUNT }}'
-
-      - name: 'Setup gcloud'
-        uses: 'google-github-actions/setup-gcloud@e30db14379863a8c79331b04a9969f4c1e225e0b' # ratchet:google-github-actions/setup-gcloud@v1
+      - name: 'Deploy Artifact Pipeline'
+        uses: 'abcxyz/pkg/.github/actions/deploy-dataflow-pipeline@main'
```
Comment (verbanicm) [inline .github/workflows/ci.yml:292]: Can we add `# ratchet:exclude` to the end of all these actions?

File: pipeline/review/review
Comment (verbanicm) [inline pipeline/review/review:1]: Can we add an ignore for this? Or is there a reason the binary is included here?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -288,127 +288,50 @@ jobs:
...
+          wif_service_account: '${{ vars.WIF_SERVICE_ACCOUNT }}'
+          image: '${{ env.DOCKER_REPO }}/commit-review-status-pipeline:${{ env.DOCKER_TAG }}-amd64'
+          template_metadata_file: './config/commit-review-status-metadata.json'
+          template_file_name: 'commit-review-status.json'
+          project: '${{ env.AUTOPUSH_PROJECT_ID }}'
+          region: '${{ env.AUTOPUSH_REGION }}'
+          bucket_name: 'github-metrics-aggreg-a-997e5e-df-store'
+          network_name: 'gma-a-df-vpc'
+          subnet_name: 'gma-a-df-central1'
+          pipeline_name: 'commit-review-status'
```
Comment (verbanicm) [inline .github/workflows/ci.yml:333]: nit: the other pipeline name is prefixed with `gma-`, can we do the same here?

File: pipeline/review/review
Comment (lock14) [reply pipeline/review/review:1]: This was included by mistake.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (lock14) [reply .github/workflows/ci.yml]: done.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (lock14) [reply .github/workflows/ci.yml]: done.

**Review by verbanicm — APPROVED** [review]

**Pull request merged**